packages/components/src/Table/index.js

```diff
@@ -1,5 +1,15 @@
+import StringFilterComponent from './Filters/StringFilterComponent';
 import Table from './Table';
 import TableCell from './Cell/TableCell';
 import TableHeaderCell from './Header/TableHeaderCell';
+import TableSortHeader from './Header/TableSortHeader';
+import ToggleFilterComponent from './Filters/ToggleFilterComponent';
 
-export { Table, TableCell, TableHeaderCell };
+export {
+	StringFilterComponent,
+	Table,
```

Comment:
Only export Table here

Reply:
but i need this components outside Table module.

Reply:
you can attach them to Table into Table/index.js

```javascript
import Table ...
import StringFilterComponent ...
Table.StringFilterComponent = StringFilterComponent

export default Table;
```